pom.xml

```diff
@@ -42,7 +42,7 @@
         <quarkus.build.skip>${skipTests}</quarkus.build.skip>
         <!-- S2i configuration -->
         <ts.global.s2i.quarkus.jvm.builder.image>registry.access.redhat.com/ubi8/openjdk-11:latest</ts.global.s2i.quarkus.jvm.builder.image>
-        <ts.global.s2i.quarkus.native.builder.image>quay.io/quarkus/ubi-quarkus-graalvmce-s2i:22.3-java17</ts.global.s2i.quarkus.native.builder.image>
+        <ts.global.s2i.quarkus.native.builder.image>registry-proxy.engineering.redhat.com/rh-osbs/quarkus-mandrel-23-rhel8:23.0-6</ts.global.s2i.quarkus.native.builder.image>
```

Comment:
I thought these are in fact ignored? https://github.com/quarkus-qe/quarkus-test-suite/pull/1330
So I think `[do not merge]` means that this one is blocked by #1330 . Ok

Reply:
They, hopefully, are, but I updated them just in case.

[do not merge] means, that after creation of this PR, I decided, that this change fit better into jenkins jobs, but here I can run the CI

Reply:
I agree, thx